l10n/sled/es-es/xml/art_installation-sleds.xml

```diff
@@ -16,6 +16,23 @@
   <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
+  <meta name="title" its:translate="yes">Guía de inicio rápido de instalación</meta>
+  <meta name="series" its:translate="no">Products &amp; Solutions</meta>
+  <meta name="description" its:translate="yes">Cómo instalar SUSE Linux Enterprise Server</meta>
```

Comment:
Needs to be 'Desktop' here and 'social-descr' is in English - will fix with the next commit